config.template.yml

```diff
@@ -98,6 +98,9 @@ log:
 ##
 ## Parameters used for TOTP generation.
 totp:
+  ## Disable TOTP.
+  disable: false
```

Comment:
I agree but this would  require people to adjust their config between minor patches unless we bump to v5 if they still wanted to use something they'd already been using. We had significant backlash in another change when doing this. Maybe it would be better to add this to the v5 roadmap?

Reply:
We can discuss this here, and apply it to both in the end.